packages/vuetify/src/components/VDatePicker/mixins/date-picker-table.ts

```diff
@@ -177,6 +177,13 @@ export default mixins(
         staticClass: 'v-date-picker-table__events',
       }, eventColors.map(color => this.$createElement('div', this.setBackgroundColor(color)))) : null
     },
+    isValidScroll (e: WheelEvent, calculateTableDate: CalculateTableDateFunction) {
+      const tableDate = calculateTableDate(e.deltaY)
+      const sanitizeType = tableDate.split('-').length === 1 ? 'year' : 'month'
```

Comment:
This split doesn't take into account for a date? What can be mentioned herewith that reason?

Reply:
Table Date will be either "yyyy-mm" (For Date Picker) or "yyyy" (For Month Picker) . When split will provide the sanitize type based upon which the min and max which is of type timestamp will be converted into the same format.

Reply:
Got it, maybe you can a single line comment on why only year and month is being used here. Rest seems good to proceed.